<launch>
    <node
    name="camera"
    pkg="arnie_vision"
    type="camera"
    args="0"
    output="screen"/>
    <node
    name="gui"
    pkg="arnie_kiosk"
    type="gui.py"
    output="screen"/>
</launch>